schema/deploy/warehouse/identifier/slices.sql

```diff
@@ -0,0 +1,13 @@
+-- Deploy seattleflu/schema:warehouse/identifier/slices to pg
+
+begin;
+
+alter table warehouse.identifier
+add column if not exists slices text[];
```

Comment:
Should `slices` be not null so that if `barcode_slices()` returns null for any reason we don't insert a bad row?

Reply:
Might as well make it not null! I've added that in the fixup .... set it to no null after populating existing rows.

Reply:
Moot now with the index (instead of `slices` column) implementation.